.github/workflows/ci.yml

```diff
@@ -8,6 +8,9 @@ concurrency:
   group: ${{ github.head_ref }}
   cancel-in-progress: true
 
+env:
+  HELM_EXPERIMENTAL_OCI: '1'
+
```

Comment:
You don't need this since your Helm version (3.11.2) supports OCI.

Reply:
Done.